crates/napi/src/threadsafe_function.rs

```diff
@@ -97,23 +98,83 @@ type_level_enum! {
 }
 
 struct ThreadsafeFunctionHandle {
-  raw: sys::napi_threadsafe_function,
+  raw: AtomicPtr<sys::napi_threadsafe_function__>,
   aborted: RwLock<bool>,
+  valid: AtomicBool,
```

Comment:
Don't need `valid` status here, just reuse `aborted` is fine

Reply:
got it